ddtrace/appsec/processor.py

```diff
@@ -92,12 +94,12 @@ class _Addresses(object):
 _COLLECTED_HEADER_PREFIX = "http.request.headers."
 
 
-def _set_headers(span, headers):
-    # type: (Span, Dict[str, Union[str, List[str]]]) -> None
+def _set_headers(span, headers, kind="request"):
```

Comment:
```suggestion
def _set_headers(span, headers, kind):
```

Doesn't seem like we need a default for this argument